packages/safe-ds-lang/src/resources/builtins/safeds/data/tabular/plotting/TablePlotter.sdsstub

```diff
@@ -14,205 +14,239 @@ from safeds.data.tabular.containers import Table
  *     val plotter = table.plot;
  * }
  */
-class TablePlotter(
-    table: Table
-) {
-    /**
-     * Create a box plot for every numerical column.
-     *
-     * @param theme The color theme of the plot. Default is "light".
-     *
-     * @result plot The box plot(s) as an image.
-     *
-     * @example
-     * pipeline example {
-     *     val table = Table({"a":[1, 2], "b": [3, 42]});
-     *     val image = table.plot.boxPlots();
-     * }
-     */
+@Category(DataScienceCategory.BasicElement)
+class TablePlotter(table: Table) {
+/**
+ * Create a box plot for every numerical column.
+ *
+ * @param theme The color theme of the plot. Default is "light".
+ *
+ * @result plot The box plot(s) as an image.
+ *
+ * @example
+ * pipeline example {
+ *     val table = Table({"a":[1, 2], "b": [3, 42]});
+ *     val image = table.plot.boxPlots();
+ * }
+ */
```

Comment:
Please keep the formatting of the remaining stub code the same. The stubs are generated mostly automatically, and we use Git patches/diffs to apply updates.

Reply:
The reformatting was not intentional. The PR with the extracted changes specifically to the Stubs fixes this.